draft-ietf-scitt-architecture.md

```diff
@@ -87,7 +87,6 @@ informative:
   I-D.draft-ietf-cose-merkle-tree-proofs: COMETRE
 
   RFC2397: DataURLs
-  RFC6024:
```

Comment:
This informative RFC is about trust anchor management and that is material to the changes being mad, did you or someone reviewing intend to remove it?

Sorry suggestions does not support adding back deleted lines anymore or I would just recommend it.

Reply:
Trust Anchor Management is a vital part of SCITT story. I suggest we should retain it, perhaps needs to be referred from the section that talks about setting the Transparency Service with necessary Trust Anchors!

Reply:
Good catch. I removed the reference as it was no longer in the doc, and called out as a make error:

The reference to RFC6024 was removed in [PR 201](https://github.com/ietf-wg-scitt/draft-ietf-scitt-architecture/pull/201/files#diff-585fb424519b79cf000445f3425fa56d328cbaca5d2a622740ddc4e5ca91dbe1L394)